docs/future-work/motor-system-improvements/reuse-hypothesis-testing-points.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Reuse Hypothesis-Testing Policy Target Points
+---
+
+The hypothesis-testing policy is able to generate candidate points on an object that, when observed, should rapidly disambiguate between similar objects, or between similar poses of the same object.
+
+Generating these points requires a model-based policy that simulates the overlap in the graphs between the two most likely objects (or the two most likely poses of the same object). This is a relatively expensive operation, and so one approach would be to store these points in long-term memory, reusing them in future episodes.
+
+For example, when we have first learn about the concept of a mug, we might need to deliberately think about the fact that its handle is what distinguishes it from many other cylindrical objects. However, once we have experienced recognizing mugs a few times, we could quickly recall that testing the handle is a good way to confirm whether we are sensing a mug, or some other object.
```

Comment:
Another way to determine ROIs on an object could be to use the evidence update as a signal. So basically locations on an object that maximally disambiguate the object from others would become hot-spots to move to next time that object is hypothesized.

Reply:
Yeah good idea!